packages/create-next-app/create-app.ts

```diff
@@ -47,16 +48,25 @@ export async function createApp({
   srcDir: boolean
   importAlias: string
   skipInstall: boolean
+  empty: boolean
 }): Promise<void> {
   let repoInfo: RepoInfo | undefined
   const mode: TemplateMode = typescript ? 'ts' : 'js'
   const template: TemplateType = appRouter
     ? tailwind
-      ? 'app-tw'
-      : 'app'
+      ? empty
```

Comment:
should "empty" imply no tailwind either? To me an empty project makes no assumptions about styling dependencies either.

Reply:
It also doesn't look like `app-tw-empty` / `default-tw-empty` is loading Tailwind.